.github/workflows/label_prs.yaml

```diff
@@ -39,20 +41,19 @@ jobs:
               - 'MODULE.*'
               - 'WORKSPACE'
               - 'bazel/**'
-              - 'common/**'
               - 'github_tools/**'
-              - 'installers/**'
-              - 'language_server/**'
               - 'proposal/scripts/**'
               - 'scripts/**'
-              - 'testing/**'
-              - 'third_party/**'
-              - 'utils/**'
             proposal:
               - added: 'proposals/*'
```

Comment:
Should this exclude proposals/scripts?

Reply:
I believe it does because it only is one `*`, so this should only match added files (or new directories) directly under `proposals`.